configs/config.yaml

```
@@ -0,0 +1,20 @@
+hyperparameters:
```

```suggestion
train:
```

I think it's a better name for this section (technically everything in this config is a hyperparameter)